_src/themes/mit/static/sass/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$red:                           rgba(163, 31, 52, 1.0);
+$dark-gray:                     rgba(138, 139, 140, 1.0);
+$light-gray:                    rgba(194, 192, 191, 1.0);
+
+$brand-primary:                 $red;
+$brand-secondary:               $dark-gray;
+$brand-accent:                  $light-gray;
```

Comment:
...and then redefine the primary, secondary, and tertiary brand colors which are used throughout the application.